test/rules/src/https_everywhere_checker/check_rules.py

```diff
@@ -96,11 +96,11 @@ def processTask(self, task):
 		problems = []
 		for url in task.urls:
 			result = self.processUrl(url, task)
-			if result:
+			if result[0]:
```

Comment:
I don't see the substantive difference here.  In `processUrl`, if result is not `None`, then it will be defined and have a `0` element.